application/common/helpers/MySqlDateTime.php

```diff
@@ -3,15 +3,21 @@
 
 class MySqlDateTime
 {
-    const MYSQL_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATETIME_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATE_FORMAT     = 'Y-m-d';
```

Comment:
@fillup I'm a little concerned about just using a date (not a specific point in time) for password expirations. Depending on how `expires_on` is interpreted (beginning of day vs. end of day) and what the server's timezone is set to (which affects how `strtotime(...)` is going to interpret these dates), we could have a nearly 2-day window between when we meant for the password to expire and when some other code using that value may actually consider it to have expired. If a potentially 2-day margin-of-error on password expirations is acceptable (and it may be, given that our servers typically default to UTC, so the error is more likely just an off-by-one margin-of-error), then I'm fine with this. If we want to reduce/remove that range of possible interpretations of the expiration date, it may be worth specifying it as a full date/time string (with timezone, ideally), such as `2018-05-24 00:00:00 UTC` (or `2018-05-24 23:59:59 UTC`, if you want end-of-day), and then just showing the relevant information to the end user.